languages/go/internal/ffi/ffi.go

```diff
@@ -45,6 +45,7 @@ func (p *PolarFfi) delete() {
 func getError() error {
 	err := C.polar_get_error()
 	errStr := C.GoString(err)
```

Comment:
Minor: could move all go string construction into a method so we don't forget it in the future.